main.go

```diff
@@ -198,6 +227,7 @@ func main() {
 	headers := make(headerSet)
 	flag.Var(&headers, "header", "HTTP request header. (can be repeated.)")
 	data := flag.String("data", "", "HTTP request data")
+	addr := flag.String("addr", ":8505", "service port to run on")
```

Comment:
I'd vote to call this flag "metric-addr" or something more specific, since it only pertains the prometheus endpoint. I wouldn't want somebody confusing this with the addr of the host to which we're sending traffic. Also, I'd prefer for this flag to default to "", and for us not to start the metrics server at all, unless the addr is explicitly provided via the flag.